packages/genesys-spark-components/src/components/stable/gux-form-field/components/gux-form-field-time-picker/gux-form-field-time-picker.tsx

```diff
@@ -87,6 +126,7 @@ export class GuxFormFieldTimePicker {
     this.setInput();
     this.setLabel();
 
+    this.labelInfo = this.root.querySelector('gux-label-info-beta');
```

Comment:
instead of querying by this should we query by the slot name `slot="label-info"`?